### ISSUE TITLE
Unfold.jl does not seem to work with Julia version 1.8.3 anymore

### ISSUE THREAD

**Issue by jschepers** [unfoldtoolbox/Unfold.jl#131]
I used Julia 1.8.3 and tried to install Unfold in a temporary environment, but 1 dependency errored and when trying to use "using Unfold" the following error concerning "UnfoldMixedModelsExt.jl" occurred (René could replicate the error.).
```
[ Info: Precompiling Unfold [181c99d8-e21b-4ff3-b70b-c233eddec679]
ERROR: LoadError: SystemError: opening file "/home/schepers/.julia/packages/Unfold/AF2na/ext/UnfoldMixedModelsExt.jl": No such file or directory
Stacktrace:
  [1] systemerror(p::String, errno::Int32; extrainfo::Nothing)
    @ Base ./error.jl:176
  [2] #systemerror#80
    @ ./error.jl:175 [inlined]
  [3] systemerror
    @ ./error.jl:175 [inlined]
  [4] open(fname::String; lock::Bool, read::Nothing, write::Nothing, create::Nothing, truncate::Nothing, append::Nothing)
    @ Base ./iostream.jl:293
  [5] open
    @ ./iostream.jl:275 [inlined]
  [6] open(f::Base.var"#387#388"{String}, args::String; kwargs::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ Base ./io.jl:382
  [7] open
    @ ./io.jl:381 [inlined]
  [8] read
    @ ./io.jl:462 [inlined]
  [9] _include(mapexpr::Function, mod::Module, _path::String)
    @ Base ./loading.jl:1484
 [10] include(mod::Module, _path::String)
    @ Base ./Base.jl:419
 [11] include(x::String)
    @ Unfold ~/.julia/packages/Unfold/AF2na/src/Unfold.jl:1
 [12] top-level scope
    @ ~/.julia/packages/Unfold/AF2na/src/Unfold.jl:62
 [13] include
    @ ./Base.jl:419 [inlined]
 [14] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt64}}, source::Nothing)
    @ Base ./loading.jl:1554
 [15] top-level scope
    @ stdin:1
in expression starting at /home/schepers/.julia/packages/Unfold/AF2na/src/Unfold.jl:1
in expression starting at stdin:1

---------------------------------------------------------------------------
JuliaError                                Traceback (most recent call last)
Cell In[20], line 1
----> 1 jl.seval("using Unfold")

File ~/.julia/packages/PythonCall/qTEA1/src/jlwrap/module.jl:25, in seval(self, expr)
     23         return ValueBase.__dir__(self) + self._jl_callmethod($(pyjl_methodnum(pyjlmodule_dir)))
     24     def seval(self, expr):
---> 25         return self._jl_callmethod($(pyjl_methodnum(pyjlmodule_seval)), expr)
     26 """, @__FILE__(), "exec"), jl.__dict__)
     27 pycopy!(pyjlmoduletype, jl.ModuleValue)

JuliaError: Failed to precompile Unfold [181c99d8-e21b-4ff3-b70b-c233eddec679] to /home/schepers/.julia/compiled/v1.8/Unfold/jl_xGUR08.
Stacktrace:
  [1] error(s::String)
    @ Base ./error.jl:35
  [2] compilecache(pkg::Base.PkgId, path::String, internal_stderr::IO, internal_stdout::IO, keep_loaded_modules::Bool)
    @ Base ./loading.jl:1707
  [3] compilecache
    @ ./loading.jl:1651 [inlined]
  [4] _require(pkg::Base.PkgId)
    @ Base ./loading.jl:1337
  [5] _require_prelocked(uuidkey::Base.PkgId)
    @ Base ./loading.jl:1200
  [6] macro expansion
    @ ./loading.jl:1180 [inlined]
  [7] macro expansion
    @ ./lock.jl:223 [inlined]
  [8] require(into::Module, mod::Symbol)
    @ Base ./loading.jl:1144
  [9] eval
    @ ./boot.jl:368 [inlined]
 [10] eval
    @ ./Base.jl:65 [inlined]
 [11] pyjlmodule_seval(self::Module, expr::Py)
    @ PythonCall ~/.julia/packages/PythonCall/qTEA1/src/jlwrap/module.jl:13
 [12] _pyjl_callmethod(f::Any, self_::Ptr{PythonCall.C.PyObject}, args_::Ptr{PythonCall.C.PyObject}, nargs::Int64)
    @ PythonCall ~/.julia/packages/PythonCall/qTEA1/src/jlwrap/base.jl:62
 [13] _pyjl_callmethod(o::Ptr{PythonCall.C.PyObject}, args::Ptr{PythonCall.C.PyObject})
    @ PythonCall.C ~/.julia/packages/PythonCall/qTEA1/src/cpython/jlwrap.jl:47
```

**Comment by behinger** [comment]
yeah, this is a problem of the Extensions introduced in Julia 1.9 - I thought I followed the guide to be backward compatible, but obviously I am not.

**Comment by behinger** [comment]
found it, I tested the julia1.8 functionality, but then changed something in the path later. Now I fixed it, found some other bugs (other functions that were not available in 1.8 any longer)

**Comment by behinger** [comment]
uff - that was more difficult than imagined. But at least on my local copy, both jl1.9 and jl1.8 worked :)

**Comment by behinger** [comment]
fixed in v0.6.3 , thanks for reporting
